vocab/properties.json

```diff
@@ -22,7 +22,7 @@
         ]
     },
     "accessibility": {
-        "description": null,
+        "description": "Level of ability to access something.",
```

Comment:
Available is not the same as accessibility. But I like the rest of what you suggested. I've entered now a mix:
"Level to which something is accessible to someone or something."

> I've added some suggestions for some of the descriptions. Some of them need to be changed because the description is wrong (e.g. handedness).

Handedness is not wrong... https://www.merriam-webster.com/dictionary/handedness

And you did not commented on it explicitly... was this a typo?

Reply:
I think you hopped over a few of my comments. There is a comment related to "handedness" directly. I'm sorry, I could have been a bit more specific with my main comment about this. It's not that the description is wrong per se, but in our context with the options available, it's more of a gradient (similar to what Andrew commented for "accessibility"). Our options are: left, right, both equally good, different one per task, both equally bad (was too lazy to look up the exact words). 
Therefore, the description "Tendency to use one hand rather than the other." is not sufficient. Also, handedness applies to feet as much as to hands, right? 
I also commented on the description for "laterality" (description: "Dominance in function or use of one of a pair of lateral homologous parts of the body."). In our context, this is not really true for "laterality" (anymore; since we split handedness and laterality now anyway) but this description would already be much better for handedness if you don't like my suggestions under handedness ;)

Reply:
The description for accessibility is nice :)